### PR TITLE
[test] don't run ansible-test sanity on a python matrix version as docker container already does that.

### PR DESCRIPTION
**Pull request by bmillemathias**
As this useless as specified by @felixfontein

### CODE REVIEW

**Comment by felixfontein** [comment]
You should remove `python` from the matrix. Otherwise you're running the same test four times :)

File: .github/workflows/ansible-test-plugins.yml [hunk 1]
```diff
@@ -26,9 +26,8 @@ jobs:
           - devel
         python:
           - 2.7
```
Comment (felixfontein) [inline .github/workflows/ansible-test-plugins.yml:28]:
```suggestion
```

File: .github/workflows/ansible-test-plugins.yml [hunk 2]
```diff
@@ -16,34 +16,31 @@ on:
...
           path: ansible_collections/gluster/gluster
 
-      - name: Set up Python ${{ matrix.python }}
+      # Just run once for the target python version
+      - name: Set up Python ${{ python-version }}
         uses: actions/setup-python@v2
         with:
-          python-version: ${{ matrix.python }}
+          # 3.8 is blocked by ansible/ansible#70155
+          python-version: 3.7
```
Comment (felixfontein) [inline .github/workflows/ansible-test-plugins.yml:39]:
```suggestion
          python-version: 3.8
```
This only matters if you don't use `--docker` in ansible-test.

**Comment by bmillemathias** [comment]
the tests will keep failing until #7 is merged

**Comment by pkesavap** [comment]
ci please test

**Pull request merged**